styles/style.css

```diff
@@ -6,3 +6,161 @@ Green: #00B172
 White: #FFF
 
 */
+
+body{
+    color: #1A1A1A;
+    background-color: #FFF;
+    margin: 0px;
+    padding: 0px;
+    font-size: 16px;
+    font-family:Verdana, Geneva, Tahoma, sans-serif
+}
+
+header {
+    display:inline-block;
+    vertical-align:top;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-top: 10px ;
+    margin-bottom: 5px;
+    margin-left: 5px;
+    margin-right: 10px;
+
+}
+
+.logo-spotify {
+    width: 275px;
+}
+
+.nav-bar{
+    display:flex;
+    align-items: center;
+    justify-content: space-evenly;
+    margin-right: 10px;
+}
+.nav-bar li {
+    display: inline-block;
+}
+
+.nav-bar ul, a{
+    font-size: 25px;
+    text-decoration: none;
+    list-style: none;
+    color: #1A1A1A;
+    margin-left: 20px;
+    font-weight: lighter;
+}
+
+.section1 {
+    background-image: url(/images/landing.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 800px;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content: center;
+    
+}
+
+.section1 > h1{
+    color: #FFF;
+    font-size: 80px;
+    font-weight: 500;
+    display: flex;
```

Comment:
no need for display: flex here.
h1 is a child element of your section1, display: flex is a property that goes to the parent container only.